third_party/github.com/bazel_json/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright 2019 Erick Johnson <ejohnson82@gmail.com>
```

Comment:
third party json parser, we can't introduce it as an http_archive since you can't patch the workspace file in a repo

Reply:
I don't think that fully explains why the vendoring is necessary, but there is an example of this upstream already so I think this is ok:
https://github.com/bazelbuild/rules_nodejs/tree/a13f2b6e2a132954fb63b7a905818cf8a6ceb824/third_party/github.com/bazelbuild/bazel-skylib/rules